### PR TITLE
Android: proper shutdown mechanism

### PR DESCRIPTION
**Pull request by paulrouget**
Fix #21834

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22021)

### CODE REVIEW

**Comment by paulrouget** [comment]
@jdm r?

File: ports/libsimpleservo/src/jniapi.rs [hunk 1]
```diff
@@ -70,6 +70,8 @@ pub fn Java_org_mozilla_servoview_JNIServo_init(
             "script::dom::bindings::error",
             // Show GL errors by default.
             "canvas::webgl_thread",
+            "compositing::compositor",
+            "constellation::constellation",
```
Comment (jdm) [inline ports/libsimpleservo/src/jniapi.rs]: Do we still want these on by default?

**Review by jdm — APPROVED** [review]

File: ports/libsimpleservo/src/jniapi.rs (hunk 1 above)
Comment (paulrouget) [reply ports/libsimpleservo/src/jniapi.rs]: yes.

**Pull request merged**